src/MoBi.Presentation/Presenter/ChartPresenter.cs

```diff
@@ -354,5 +355,10 @@ public override void ReleaseFrom(IEventPublisher eventPublisher)
          displayPresenter.DragOver -= OnDragOver;
          Clear();
       }
+
+      public void Handle(ObservedDataRemovedFromAnalysableEvent e)
+      {
+         Refresh();
```

Comment:
hum. you will refresh all the time even if the chart has nothing to do with the event. I think this is not ok